src/components/Apps/AccountListTable/AccountList.vue

```diff
@@ -388,6 +390,7 @@ export default {
     }
   },
   computed: {
+    ...mapGetters(['currentUserIsSuperAdmin']),
     accountCreateUpdateTitle() {
       if (this.addTemplate) {
         return this.$t('AddAccountByTemplate')
```

Comment:
I do not see any major issues or potential improvements in the given code snippet. The code seems to be well-written and functional based on my knowledge of Vue.js at the time I generated this response.

If you want specific optimizations or modifications suggested for improving performance, maintainability, or adding features, please provide more details about what exactly you need assistance with.